README.md

```diff
@@ -31,49 +34,60 @@ to Install all the stack, you should execute into the root folder the line below
 docker-compose -f "docker-compose.yml" up -d --build
 ```
 
-ps: If you have this kind of error “no matching manifest for windows/amd64 in the manifest list entries”, switch to Linux container in Docker Desktop.
+P.S. If you have the kind of error "no matching manifest for windows/amd64 in the manifest list entries", switch to **Linux container** in Docker Desktop.
+
 ## Usage
 
-### Orthanc server
+### Orthanc Server
 
-Once Orthanc is running, use Mozilla Firefox at URL [http://localhost:8042/](http://localhost:8042/) to interact with Orthanc. The default username is `barbecuehealth` and its password is `barbecuehealth`.
+Once Orthanc is running, use Mozilla Firefox at URL <http://localhost:8042/> to interact with Orthanc. The default username is `barbecuehealth` and its password is `barbecuehealth`.
 
-If you want to change login/password, edit file `config/orthanc.json` and add your new login into RegisteredUsers
+If you want to change the login/password, edit the file `config/orthanc.json` and add your new login into `RegisteredUsers`.
 
 ### Mongo Database
 
-A web Mongo client is available on the stack, you just need to use [http://localhost:8043/](http://localhost:8043/) to access MongoDB.
+A web Mongo client is available on the stack, you just need to use <http://localhost:8043/> to access MongoDB.
+
+### Web Client
+
+It is developed in React TypeScript, and Yarn is used as the package manager/CLI.
+
+#### Requirement
+
+To develop, we need following tools:
 
-### Web client
+- [Node.js](https://nodejs.org/en/) >= 12
+- [Yarn](https://classic.yarnpkg.com/lang/en/) ~1.20
 
-It is developed in React TypeScript and Yarn is using as package manager/cli
+On macOS, you can install those tools with Homebrew by running the command `brew install yarn`.
```

Comment:
On macOS, install these tools can be installed using Homebrew, by running the command `brew install yarn`.